includes/admin.form.inc

```diff
@@ -101,7 +101,7 @@ function islandora_audio_admin_form_lame_ajax_callback(array $form, array $form_
  */
 function islandora_audio_vbr_quality_validate(array $element, array &$form_state) {
   if (!empty($element['#value'])) {
-    if ($element['#value'] < 0 || $element['#value'] > 9.99) {
+    if ($element['#value'] < 0 || $element['#value'] > 9.999) {
```

Comment:
@rosie, since PHP is not type strict, and you need to check for a specific number i would suggest to cast #value to a number with decimals to make sure? Also
 > Please enter a VBR quality value between 0 and 9.999.

could be easier the read the other way around:assume it's wrong(form error) , make it pass if >0 and < 10